cli/tools/test.rs

```diff
@@ -827,7 +827,6 @@ fn extract_files_from_regex_blocks(
 }
 
 fn extract_files_from_source_comments(
-  current_dir: &Path,
```

Comment:
Great to see this removed - I presume it's because we're sure all the specifiers are absolute?

Reply:
This was unnecessary. `specifier` is a `ModuleSpecifier` here: https://github.com/denoland/deno/pull/18294/files#diff-30ed960712f694079cb9a92d07667599562791c866e47e45410bddde238fac84L803-R808